cookbooks/bcpc-hadoop/attributes/mapred.rb

```diff
@@ -10,15 +10,17 @@
 default[:bcpc][:hadoop][:mapreduce][:site_xml].tap do |site_xml|
   site_xml['mapreduce.admin.map.child.java.opts'] =
     '-server -Djava.net.preferIPv4Stack=true -Dhdp.version=' +
-    node[:bcpc][:hadoop][:distribution][:release].to_s
+    '${hdp.version}'
```

Comment:
Do we not need to pass `hdp.version` in on the command line still? I don't see `hdp.version` (or a fraction there of) in my ignorant use of `strings` in `/usr/hdp/current/hbase-regionserver/lib` on our machines.